render/mapping.go

```diff
@@ -441,8 +441,10 @@ func MapProcess2Container(n RenderableNode, _ report.Networks) RenderableNodes {
 		id = MakeContainerID(containerID)
 		node = NewDerivedNode(id, n)
 	} else {
+		nCopy := n.Copy()
+		nCopy.Node = nCopy.Node.WithID("").WithTopology("") // Wipe the ID so it cannot be rendered.
```

Comment:
It removes the red section.
<img width="440" alt="screen shot 2016-02-12 at 13 28 35" src="https://cloud.githubusercontent.com/assets/250199/13008184/cba26546-d18c-11e5-8846-c429505920ae.png">